frontend/src/metabase/metadata/pages/DataModel/components/PreviewSection/ObjectDetailPreview.tsx

```diff
@@ -58,15 +69,36 @@ const ObjectDetailPreviewBase = ({
     return <Error message={error} />;
   }
 
-  if (!data || !zoomedRow) {
+  if (!data || !row || columns.length === 0) {
     return (
       <Stack h="100%" justify="center" p="md">
         <EmptyState title={t`No data to show`} />
       </Stack>
     );
   }
 
-  return <Visualization rawSeries={rawSeries} />;
+  return (
+    <Stack gap={0} p="lg">
+      {headerColumns.length > 0 && (
+        <Box pb="md" pt="xs">
+          <Box ml={rem(-8)}>
+            <Header columns={columns} icon={icon} row={row} />
+          </Box>
+        </Box>
+      )}
+
+      {columns.length - headerColumns.length > 0 && (
```

Comment:
Now we show all columns in `DetailsGroup` (#63889), so this line should be:

```suggestion
      {columns.length > 0 && (
```

Reply:
Can you please also update this line in `DetailViewSidesheet`? (My bad, I forgot :speak_no_evil:)